exasol_advanced_analytics_framework/udf_communication/messages.py

```diff
@@ -79,6 +80,7 @@ class SynchronizeConnection(BaseMessage, frozen=True):
     message_type: Literal["SynchronizeConnection"] = "SynchronizeConnection"
     source: ConnectionInfo
     destination: Peer
+    attempt: int
```

Comment:
Maybe also rename it like in exasol_advanced_analytics_framework/udf_communication/peer_communicator/background_thread/connection_closer/close_connection_sender.py
`attempt_count` or `attempt_seq_number` or `attempt_idx`

Reply:
Hm, attempt seems perfect for me. Count is something absolute, so this doesn't fit. It is also not really an idx number. Sequence number would most, but has usually also different meaning.